frontend/src/components/modals/AddTaskModal.tsx

```diff
@@ -58,7 +58,7 @@ export const AddTaskModal: Modal<ModalTypes.ADD_TASK_MODAL> = ({
       };
 
       try {
-        const res = await addTaskTrigger({
+        const task = await addTaskTrigger({
           roadmapId: chosenRoadmapId!,
           task: req,
         }).unwrap();
```

Comment:
I noticed that the addTaskTrigger returns a task with no property `ratings` at all when the request has no ratings (or the ratings is an empty array) since server returns `task: ..., ratings: undefined` in that case. Should this be looked at deeper, or this the `task: { ...task, ratings: [] },` fix enough for now? Currently RateTaskModal can't handle a task with no ratings property.